sdk/docker/dockerSkeleton/Dockerfile

```diff
@@ -0,0 +1,65 @@
+#
+# Dockerfile for building a blackbox container with a nodejs server interfacing to a client binary.
+#
+FROM ubuntu:14.04
+
+ENV DEBIAN_FRONTEND noninteractive
+
+# Initial update and some basics.
+# This odd double update seems necessary to get curl to download without 404 errors.
+RUN apt-get update --fix-missing && \
+apt-get install -y wget && \
+apt-get update && \
+apt-get install -y curl && \
+apt-get update
+
+# Get gcc
+RUN apt-get update --fix-missing && \
+apt-get -y install gcc
+
+# install nodejs and npm
+RUN curl -sL https://deb.nodesource.com/setup_0.12 | bash - && \
+apt-get install -y nodejs
+
+# install common packages
+RUN npm install -g body-parser@1.12.0 \
```

Comment:
Probably yes, I did not look into the app, only moved it. Likely outdated some of the stuff.